subsys/bluetooth/host/classic/a2dp.c

```diff
@@ -196,7 +194,7 @@ static int a2dp_set_config_ind(struct bt_avdtp *session, struct bt_avdtp_sep *se
 
 	*errcode = 0;
 
-	__ASSERT(sep, "Invalid sep");
+	BT_ASSERT_MSG(sep, "Invalid sep");
```

Comment:
What's the reason for changing to `BT_ASSERT()`? My preference would be that we completely remove that API from the Bluetooth subsystem and only use the Zephyr-wide assert API.

Reply:
Updated all assert